### ISSUE TITLE
License in Cargo.toml should be BSD-2-Clause AND ISC

### ISSUE THREAD

**Issue by ignatenkobrain** [xiph/rav1e#2181]
Hello,

I've noticed that https://github.com/xiph/rav1e/blob/master/src/ext/x86/x86inc.asm is under ISC license, but Cargo.toml does not reference that. Would be very nice if you could correct that.

Thanks in advance!

**Comment by rzumer** [comment]
I'm not sure that rav1e needs to include the licenses of every component it uses in its manifest. Otherwise we should be looking at the licenses of every dependency, and include those as well. ISC is not a copyleft license.

**Comment by ignatenkobrain** [comment]
@rzumer of course, you don't need to include license of all components you use. But this file is part of rav1e itself, so that makes rav1e be not BSD but BSD AND ISC.

**Comment by rzumer** [comment]
x264 is licensed under the GPL, you can see that in its repository [here](https://code.videolan.org/videolan/x264/) (see "COPYING"). It's not "GPL and ISC", it's (as a whole) GPL and happens to contain one file licensed as ISC. Similarly rav1e is BSD-licensed as a whole even if it includes code licensed under ISC and MIT.
